webserver/webserver/src/main/java/io/helidon/webserver/HttpInitializer.java

```diff
@@ -131,6 +133,23 @@ void queuesShutdown() {
         });
     }
 
+    void updateSslContext(String socketName, SslContext context) {
+        if (sslContext == null) {
+            if (context != null) {
+                throw new IllegalStateException("TLS was not configured on the socket " + socketName + " before. Could not be "
+                                                        + "enabled at runtime.");
+            }
+            //Do not set anything
+        } else if (context == null) {
+            //Disabling TLS context is not possible at runtime
```

Comment:
Here you mean "TLS was **not** configured" right ?
Maybe just mirror the message from the previous branch: "Enabling TLS context at runtime is not allowed". Enable implies that it is currently disabled.

Reply:
Reworked